index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.0.0/dist/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+  <header id="header">
+    <img src="images.png" alt="image logo " id="header-img" width="50px">
+    <nav id="nav-bar">
+      <a href="#features" class="nav-link">Features</a>
+      <a href="#how_it_works" class="nav-link">How It works</a>
+      <a href="#pricing" class="nav-link">Pricing</a>
+    </nav>
+
+  </header>
+  <h1>Hello World, we are software developers!</h1>
+
+  <section id="features">
+    <iframe id="video" width="300" height="300" src="https://www.youtube.com/embed/50kH47ZztHs" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
video is on top of the navbar, so you should create a flexbox value on it so you can separate it from the others element in the project.